backends/p4tools/modules/testgen/targets/bmv2/bmv2.h

```diff
@@ -7,13 +7,31 @@
 
 namespace P4Tools::P4Testgen::Bmv2 {
 
+/// Extends the CompilerResult with the associated P4RuntimeApi
+class BMv2V1ModelCompilerResult : public CompilerResult {
+ private:
+    /// The runtimeAPI inferred from this particular BMv2 V1Model P4 program.
+    P4::P4RuntimeAPI p4runtimeApi;
```

Comment:
Is this not used at this point?

Reply:
Yes, I think this is an artifact of the split. I could try to move this into the children PRs.

Reply:
While it would be better not leaving this here, I could also live with it merged in this PR.